integration_test/functions/src/index.ts

```diff
@@ -7,6 +7,7 @@ import { Request, Response } from 'express';
 export * from './pubsub-tests';
 export * from './database-tests';
 export * from './auth-tests';
+let numTests = 4;  // Make sure to increase this as you add tests!
```

Comment:
The tests are not actually not quite the same as the number of promises, but they _are_ the same as the number of exported functions - so I've gone off that. :)